src/quo2/components/wallet/network_routing/animation.cljs

```diff
@@ -0,0 +1,116 @@
+(ns quo2.components.wallet.network-routing.animation
+  (:require [react-native.reanimated :as reanimated]))
+
+(def ^:private slider-timing 300)
+
+(defn show-slider
+  [opacity-shared-value]
+  (reanimated/animate opacity-shared-value 1 slider-timing))
+
+(defn hide-slider
+  [opacity-shared-value]
+  (reanimated/animate opacity-shared-value 0 slider-timing))
+
+(defn increase-slider
+  [width-shared-value height-shared-value]
+  (reanimated/animate width-shared-value 8 300)
```

Comment:
why 300 is not private slider-timing here? 🤔

Reply:
it was a mistake! :facepalm:  Fixed now! :+1: